src/main/java/listeners/VideoRecorder.java

```diff
@@ -0,0 +1,87 @@
+package listeners;
+
+import org.monte.media.Format;
+import org.monte.media.math.Rational;
+import org.monte.screenrecorder.ScreenRecorder;
+import webdriver.DriverManager;
+import ws.schild.jave.*;
+
+import java.awt.*;
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static org.monte.media.FormatKeys.*;
+import static org.monte.media.VideoFormatKeys.*;
+
+public class VideoRecorder {
+    private final String RECORD_DIRECTORY = "videos\\";
+    private ScreenRecorder screenRecorder;
+    File newFileName = null;
```

Comment:
Why is it a global variable and why is it initialized with a null? Try to avoid changeable global variables, that's a bad practice. You made the same mistake several times